.gitignore

```diff
@@ -15,3 +15,5 @@ dist
 /RUNNING_PID
 /.settings
 *.iws
+node_modules/
```

Comment:
hi @rpowis Is the change to `.gitignore` of this project needed as well? It shouldn't be generating `node-modules` anyway

Reply:
I'm also wondering if it's worth including `npm-debug.log`, these are commonly incorrectly checked in.

As a lower priority we could also ignore `yarn-debug.log` and `yarn-error.log` for those who use YARN - a lower priority because people who opt-in to using non-standard tools can/should take responsibility for these things as we're not going to support every non-standard tool.

Reply:
Seeing the whole changelist `npm-debug.log` is there and we have no known user need for yarn (we're not currently endorsing its use) so I'd say this is a good and complete change.

Reply:
@konradwudkowski Yes true. I just did grepped the repo and added it to everywhere that popped up assuming one of the files was tests. Serves me right for multitasking 😞

@matcarey I already added `npm-debug.log` but was reluctant to add yarn files for that very reason. We don't explicitly recommend (or even mention) using yarn anywhere and all our guidance uses `npm` in command examples.

I guess it wouldn't hurt to add yarn stuff as it wouldn't be in the root `.gitignore` too now. The GOV.UK prototype kit docs might one day recommend it, in which case we'd already be covered.